ballerina/tests/test.bal

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/test;
+import ballerina/log;
+configurable string token = ?;
+// Initialize the client with auth configuration
+ConnectionConfig config = {
+    auth: {
+        token: token
+    }
+};
+final Client intercom = check new(config);
+@test:Config {}
+function getAdminTest() returns error? {
+    
+    AdminList|error response = intercom->/admins;
+    if response is error {
+        log:printError("Failed to get Admin", response);
+        test:assertFail("API call failed: " + response.message());
+    }
+    log:printInfo("Test passed: Successfully retrieved admin data");
```

Comment:
Shall we add a test assertion here to check if the received response has a value?
ex: https://github.com/ballerina-platform/module-ballerinax-twitter/blob/04c928d11d137bbae04aba171dcdcb20d1021afc/ballerina/tests/tests.bal#L41